docs/issue-3103-analyse.md

```
@@ -0,0 +1,128 @@
+
+### Analyse ORBIS (Section 4)
+
+| Métrique | Valeur |
+|----------|--------|
+| Total profils ORBIS | 785 |
+| Profils avec doublons (sur 500 analysés) | 286 (57%) |
+| Même `external_id` | 221 |
+| **`external_id` différent** | **145** ← Cause confirmée |
+
+Exemple concret :
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)